### PR TITLE
Styling adjustments to align entry detail buttons during edit and view mode change

### LINKED ISSUE #480: DR: Refine the entry details to be better aligned to photon
*The read-only, entry details view of the editor*

**Inivision: https://mozilla.invisionapp.com/share/W3EU997YN#/270503568_1-1-1_Lockbox_Home_-_Focus_Change**

**Zeplin: https://zpl.io/an6g71l**

## Tasks ##
- [x] Vertical padding between fields to match the create/edit inputs seamlessly
- [x] ~~Vertical padding between header and content~~ -Included with #479 
- [x] Styling of password dots in read-only state (Fira Monospace)
- [x] ~~Update input background to #ffffff~~ -Included with #479

### PR DESCRIPTION
**Pull request by nickbrandt**
Resolves #480 

-~Modified password read only char code to match design~
-Fixed issue where bottom buttons would jump a bit from edit to view state

### CODE REVIEW

**Comment by devinreams** [comment]
This is based off #567 so only work starting at commit 786c93e095d32ab836977dce2a87a522b6f0264b is new.

**Comment by m8ttyB** [comment]
I can confirm that this works in Win10 on Firefox Nightly.

**Comment by jimporter** [comment]
> I can confirm that this works in Win10 on Firefox Nightly.

Does this mean "the dots when you're viewing a password and when you're editing a password" are identical? That's the main issue I expect to see on Windows. Screenshots would be nice for confirmation. :)

**Comment by m8ttyB** [comment]
@jimporter good point, no. The size of the dots change. I was trying to put together a screencast, took me a bit. 

pwds

**Comment by devinreams** [comment]
The "obfuscated" password dots are inconsistent switching from view/edit mode and changing this one makes it consistent on Mac but even more inconsistent on Windows and Linux. We should undo this change here and see about coming back to this at #615 since we've agreed we'll prefer/prioritize the ~95% on Win.

@nickbrandt do you mind pulling that dot change out? just discussed it with @jimporter and @changecourse.

The remainder of this fix ("bottom buttons would jump a bit from edit to view state") should be good, though. 🙃

**Comment by nickbrandt** [comment]
@devinreams @jimporter password dots reverted. Should be good to go!

**Comment by m8ttyB** [comment]
Here's the behavior on win10, Firefox nightly.

win10_pwds_reverted

File: src/webextension/list/components/item-fields.css [hunk 1]
```diff
@@ -32,3 +32,11 @@
 .notes {
   min-height: 60px;
 }
+
+.notes-read-only {
+  min-height: 73px;
+}
+
+.password {
+  font-weight: normal;
```
Comment (jimporter) [inline src/webextension/list/components/item-fields.css]: I *think* this is unnecessary now (at least on Linux); if it's actually required to make things look ok on Windows though, then I'm fine with keeping it.

**Comment by devinreams** [comment]
OK this PR looks bigger than it is becuase it was based on a branch that is now merged. Looking at the ["Files changed" view](https://github.com/mozilla-lockbox/lockbox-extension/pull/608/files) shows this is just changing the height of notes so the buttons beneath don't move around too much.

@jimporter @m8ttyB would you like to take one final look at this small change before we merge (by) tomorrow?

**Comment by jimporter** [comment]
@devinreams I'm 99% likely to r+ this; then I'll rebase and merge. Hopefully within the hour.

**Comment by devinreams** [comment]
@jimporter thanks for cleaning up (and ultimately merging) this branch 😍

**Pull request merged**